app/components/captcha_submit_button_component.rb

```diff
@@ -19,12 +19,12 @@ def show_mock_score_field?
   def recaptcha_script_src
     return @recaptcha_script_src if defined?(@recaptcha_script_src)
     @recaptcha_script_src = begin
-      if IdentityConfig.store.recaptcha_site_key_v3.present?
+      if IdentityConfig.store.recaptcha_site_key.present?
         UriService.add_params(
           recaptcha_enterprise? ?
             'https://www.google.com/recaptcha/enterprise.js' :
             'https://www.google.com/recaptcha/api.js',
-          render: IdentityConfig.store.recaptcha_site_key_v3,
+          render: IdentityConfig.store.recaptcha_site_key,
         )
       end
     end
```

Comment:
oh also as long as we're here, the `begin...end` block is redundant. an `if` block will return nil if the condition is false

Reply:
Hm, that requires (by linter) pretty funky indentation to implement, I think I'd prefer it as is, or maybe I could shift the code around to make the condition an early (non-memoized) return?

```rb
  def recaptcha_script_src
    return @recaptcha_script_src if defined?(@recaptcha_script_src)
    @recaptcha_script_src = if IdentityConfig.store.recaptcha_site_key.present?
                              UriService.add_params(
                                recaptcha_enterprise? ?
                                  'https://www.google.com/recaptcha/enterprise.js' :
                                  'https://www.google.com/recaptcha/api.js',
                                render: IdentityConfig.store.recaptcha_site_key,
                              )
                            end
  end
```

Reply:
Requires an extra line break, but found a workable solution in 5ec9368101 via https://github.com/rubocop/ruby-style-guide/issues/179 + https://github.com/rubocop/ruby-style-guide#indent-conditional-assignment

Reply:
oh that's great! my next idea was a trailing `if`, but those are too sneaky for their own good

```ruby
    @recaptcha_script_src = UriService.add_params(
                              recaptcha_enterprise? ?
                                'https://www.google.com/recaptcha/enterprise.js' :
                                'https://www.google.com/recaptcha/api.js',
                              render: IdentityConfig.store.recaptcha_site_key,
                            ) if IdentityConfig.store.recaptcha_site_key.present?
```

Reply:
Yeah I think I saw a different lint error for that style 😅